vector/src/main/java/im/vector/app/features/crypto/recover/BootstrapSharedViewModel.kt

```diff
@@ -275,6 +313,22 @@ class BootstrapSharedViewModel @AssistedInject constructor(
         }
     }
 
+    private suspend fun clearAllExistingBackups() {
```

Comment:
Could we move this code into a dedicated use case `ClearAllExistingBackupsUseCase`?